test_stopwatch_global.py

```diff
@@ -20,27 +20,35 @@ def global_sw_fixture(request):
 
 @pytest.mark.usefixtures('global_sw_fixture')
 class TestStopWatchGlobal(object):
+    def test_global_sw(self):
```

Comment:
Sorry for moving this around too much, but this seems to be the correct place for it. :)